twine-upload.sh

```diff
@@ -46,7 +46,8 @@ if [[ "${INPUT_USER}" == "__token__" && -z "${INPUT_PASSWORD}" ]] ; then
     echo \
         '::notice::Attempting to perform OIDC credential exchange' \
         'to retrieve a temporary short-lived API token for authentication' \
-        "against ${INPUT_REPOSITORY_URL}"
+        "against ${INPUT_REPOSITORY_URL} due to __token__ username with no " \
```

Comment:
`echo` binary in `shell` concatenates separate arguments with whitespaces. So this would produce a double whitespace as in #140.
```suggestion
        "against ${INPUT_REPOSITORY_URL} due to __token__ username with no" \
```